sbws/lib/relaylist.py

```diff
@@ -288,6 +289,12 @@ def increment_relay_recent_priority_list_count(self):
             self.relay_recent_priority_list_count = 0
         self.relay_recent_priority_list_count += 1
 
+    def is_old(self):
+        """Whether the last consensus seen for this relay is older than the
+        measurement period.
+        """
+        timestamp.is_old(self.last_consensus_timestamp)
```

Comment:
Shouldn't this "return" the value from `is_old()` ?

Reply:
auch, totally